js/src/pages/create-paid-ads-campaign/index.js

```diff
@@ -146,9 +147,16 @@ const CreatePaidAdsCampaign = () => {
 							content: (
 								<AdsCampaign
 									trackingContext={ eventContext }
-									onContinue={ () =>
-										handleContinueClick( STEP.ASSET_GROUP )
-									}
+									continueButton={ ( props ) => (
+										<ContinueButton
+											{ ...props }
+											handleContinueClick={ () =>
```

Comment:
The `handleContinueClick` should be updated accordingly after [renaming](https://github.com/woocommerce/google-listings-and-ads/pull/2577/commits/2564a42648754191c155cb8d53e2ad9ac3df25a9#diff-6b724af416790c2e621567319e6734b68b00681465f87e9e0e710dda87aa6870R19).

Reply:
My mistake, I did make the changes but the push didn't go through earlier. Sorry, for that.
 Now it should reflect to onClick.